src/main/kotlin/com/example/ticketing_service/queue/presentation/QueueController.kt

```
@@ -0,0 +1,50 @@
+        // 저장소에 등록
+        sseRepository.save(userId, emitter)
+
+        // 연결 종료/에러/타임아웃 시 삭제하는 콜백 등록
+        emitter.onCompletion { sseRepository.delete(userId) }
+        emitter.onTimeout { sseRepository.delete(userId) }
+        emitter.onError { sseRepository.delete(userId) }
+
+        // 최초 연결 시 더미 데이터 전송 (503 에러 방지용)
+        try {
+            emitter.send(SseEmitter.event().name("connect").data("Connected"))
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)